### ISSUE TITLE
Offer read-only tasks list

### ISSUE THREAD

**Issue by dilyanpalauzov** [dmfs/opentasks#1042]
A CalDAV server can expose read-only collections, which contain Tasks (VTODO).  DAVx⁵ can detect read-only CalDAV collections, but apparently it does not say to OpenTasks, that the collection is read-only. In turn the user can create tasks in the task list within OpenTasks.  The changes are however lost on the next synchronization.

• Create in OpenTasks a notion of a read-only task list and a mechanism, by which DAVx⁵ can communicate to OpenTasks the writable status of the collection.  Do not allow users to modify, create or delete tasks from Read-Only tasks list.  Optionally hide read-only tasks list with no tasks included (or provide this as an option).

**Comment by ilippert** [comment]
I would very much like to see read-only task lists supported

**Comment by ilippert** [comment]
Of course, this is an old issue - see https://github.com/dmfs/opentasks/issues/851 - Maybe @dmfs, close this issue in favour of the #851 issue?
